app/javascript/components/Pages/Home.js

```diff
@@ -1,10 +1,53 @@
 import React, { Component } from "react"
+import { Row, Col } from "reactstrap"
+import { Link } from 'react-router-dom'
 
 export default class Home extends Component {
-    render(){
-        return(
+    render() {
+        return (
             <>
+
                 <h2>Home</h2>
+                <br></br>
```

Comment:
Instead of br tags, you can also try creating css styles to do padding on the bottom.